storage/test/parser/parse_perfdata.cc

```diff
@@ -29,6 +30,12 @@ using namespace com::centreon::broker;
 // When parse_perfdata() is called with a valid perfdata string
 // Then perfdata are returned in a list
 TEST(StorageParserParsePerfdata, Simple1) {
+  try {
```

Comment:
Why is this needed ?

Reply:
In case you don't execute all the tests. If you execute all the tests but not Lua ones, the initialization may fail.

Reply:
Could you give me more details ? This behavior does not seem legit.